src/content/development/building-testing/ci-maintenance/_index.en.md

```diff
@@ -22,3 +22,14 @@ Basic Kubernetes Support | All non-latest supported versions | One defaults-only
 Full Kubernetes Support | All non-latest supported versions | Workflow to run all CI for all of the currently-supported, non-latest Kubernetes versions. Run periodically, on releases, or manually by adding the `e2e-all-k8s` label.
 Unsupported Kubernetes Cut-off | Oldest working version | One defaults-only E2E for the oldest Kubernetes version known to work with Submariner. This tests the cut-off version used by `subctl` to prevent installing Submariner in environments that are known to be unsupported.
 <!-- markdownlint-enable line-length -->
+
+## Shflags Library Version
+
+Shipyard uses the shflags library. Because it must be downloaded (if missing) and sourced at the start of Shipyard's shared scripts, the
+version is maintained in a special place and must be manually updated.
+
+```shell
+SHFLAGS_VERSION=${SHFLAGS_VERSION:=<version>}
+```
+
+[submariner-io/shipyard/blob/devel/scripts/shared/lib/shflags](https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/lib/shflags)
```

Comment:
```suggestion
[submariner-io/shipyard/scripts/shared/lib/shflags](https://github.com/submariner-io/shipyard/blob/devel/scripts/shared/lib/shflags)
```

I didn't mean to include these elements, they are leftover from the URL I copied. Wanted a path consistent with https://submariner.io/development/building-testing/ci-maintenance/#shipyard-base-image-software. Hoping that's a trivial change that doesn't need re-review.